rn/Teacher/ios/Teacher/TeacherTabBarController.swift

```diff
@@ -110,4 +115,11 @@ extension TeacherTabBarController: UITabBarControllerDelegate {
 
         return true
     }
+
+    @objc private func checkForPolicyChanges() {
+        let env = AppEnvironment.shared
+        env.checkAcceptablePolicy(from: self, cancelled: {
+                AppEnvironment.shared.loginDelegate?.changeUser()
```

Comment:
There seems to be an extra tab here. Also, we could use the env helper in the block as well.